nncf/quantization/algorithms/weight_compression/scale_estimation.py

```diff
@@ -165,8 +165,6 @@ def apply(
             original_weight = fns.zeros_like(weight) + weight
 
             compressed_weights, scale, zp = do_integer_quantization(original_weight, reduction_axis, config)
-            zp = zp.astype(scale.dtype)
```

Comment:
if zp is not None:
    zp = zp.astype(scale.dtype)
this conversion is important for performance